tests_disabled.json

```diff
@@ -161,10 +161,6 @@
             "name" : "selftests.test_framework.ExampleTest.test_deproxy_srvclient",
             "reason" : "Disabled by issue #73"
         },
-        {
-            "name" : "http2_general",
-            "reason" : "Disabled by issue #261"
```

Comment:
I believe would be better to unmask whole http2_general after fixing https://github.com/tempesta-tech/tempesta/issues/1669. Headers tests falls time to time in random order.